src/Model/Behavior/ShadowTranslateBehavior.php

```diff
@@ -276,10 +279,14 @@ public function beforeSave(Event $event, EntityInterface $entity, ArrayObject $o
         $fields = array_keys($values);
         $primaryKey = (array)$this->_table->primaryKey();
```

Comment:
Our discussion on irc about expanding the scope of this PR (by about 5%) so that it allows for multi-field primary keys remains unactioned.